moduleroot/spec/spec_helper.rb.erb

```diff
@@ -1,5 +1,12 @@
 require 'puppetlabs_spec_helper/module_spec_helper'
 require 'rspec-puppet-facts'
+
+begin
+  require 'spec_helper_local'
+rescue LoadError => loaderror
+  puts "Could not require spec_helper_local: #{loaderror.message}"
```

Comment:
`s/puts/warn/` so that this is printed to stderr and can be disabled with other warnings. Also, I'd recommend guarding this to only attempt the require if `spec/spec_helper_local.rb` exists, otherwise it'll print an unnecessary error for people who don't need/use a spec_helper_local

Reply:
@rodjek updated with suggestions.